test/account/deposit.ts

```diff
@@ -19,31 +20,38 @@ describe('Deposit Tests', () => {
     clusterResult1 = await helpers.registerValidators(4, 1, minDepositAmount, helpers.DataGenerator.cluster.new(), [GasGroup.REGISTER_VALIDATOR_NEW_STATE]);
   });
 
-  it('Deposit as owner emits FundsDeposit event', async () => {
+  //Owner deposit
+  it('Owner deposits emits "FundsDeposit"', async () => {
```

Comment:
owner is not relevant, keeping the standard we will use the function name in the test:
"Deposit function emits "FundsDeposited""